src/main/resources/application.yml

```diff
@@ -2,6 +2,8 @@ server:
   port: 9092
   remote:
     uri: localhost:9090
+  debug: false
+
```

Comment:
If this is only used by R&D do not expose it to everybody.
```suggestion
```

Reply:
It is still a functionality that can be used by anyone, especially if we improve on it.

Since we are not giving a jar but the repo itself for people to use MAC, setting it in DebugVisitor will still expose it to everyone